src/mono/mono/mini/seq-points.c

```diff
@@ -269,8 +269,7 @@ mono_get_seq_points (MonoMethod *method)
 		mono_error_assert_ok (error);
 	}
 
-	// FIXME:
-	jit_mm = get_default_jit_mm ();
+	jit_mm = jit_mm_for_method (method);
 	jit_mm_lock (jit_mm);
 	seq_points = (MonoSeqPointInfo *)g_hash_table_lookup (jit_mm->seq_points, method);
 	if (!seq_points && method->is_inflated) {
```

Comment:
inside this `if`, we also look up `declaring_generic_method` and `shared_method`.
Are those guaranteed to be in the same memory manager as `method`?  /cc @vargaz